### DISCUSSION TITLE
Test and demonstrate scheduling functionality using PlanDefinition, CarePlan, and Task

### DISCUSSION THREAD

**Discussion by f-odhiambo** [opensrp/fhircore#532] [Ideas]
In ANC tasks will be created from this [schedule] done a while back by Carolyn in 2019 (https://docs.google.com/spreadsheets/d/1dx7F5NMCsx5SPHPsSlj6A1r1EGI7nzTgkfabgz7v0Yc/edit?usp=sharing)

Data requirements: We need to pass 2 variables when evaluating each task. 
1. Gestation age(GA). Since this can be calculated in many ways in the ANC. we can expect the event or client has the gestation age or Expected delivery date (EDD) which can be used to calculate GA. 
3. The date the first contact was made. This will be used to evaluate if rule 7 or 1-6 should be applied for the woman as per the schedule shared above.

We may need a plan definition that will be used to trigger this e.g 

**Subtasks**
- [ ] #283 
- [ ] Implementation on Android FHIR SDK - core functionliaty/foundation
- [ ] Implementation on FHIRCore - tweeks

**Comment by pld** [comment]
I don't think we should worry about implementing the specific logic of the linked schedule until we confirm this is the schedule we will be using something simpler but demonstrative that we can support the more complex schedule linked is sufficient.

Going to update the title to clarify this is demonstrating and testing the functionality of scheduling, not necessarily implementing it

**Comment by pld** [comment]
Relatedly, I don't know what if anything would be needed in Android FHIR SDK for this
